NEWS.md

```diff
@@ -4,6 +4,15 @@
   aesthetics they accept: `xmin_final`, `xmax_final`, `xlower`,
   `xmiddle` and `xupper` are now valid `x` aesthetics.
 
+* `ggtitle` and `labs` now take a `subtitle` parameter which makes it 
+  possible to add a subtitle below the main plot title.
+  
+* `labs` now takes an `annotation` parameter that will set the label for 
+  annotation text below the plot panels
+  
+* the main plot title and subtitle are left-justified (`hjust = 0`) by
+  default, now
```

Comment:
You can drop ", now".

Can you please make it a proper sentence with leading capital and trailing period?